.eslintrc.js

```diff
@@ -96,10 +95,8 @@ module.exports = {
       WARNING,
       {disallowTypeAnnotations: false},
     ],
-    'import/first': OFF,
     'import/order': OFF,
     'import/prefer-default-export': OFF,
-    'lines-between-class-members': OFF,
```

Comment:
I'd prefer to keep this off. As you see, we rarely use classes, and when we do, we have lots of one-line class members that are not worth adding paddings.